setup/start.sh

```diff
@@ -106,7 +106,13 @@ source setup/dkim.sh
 source setup/spamassassin.sh
 source setup/web.sh
 source setup/webmail.sh
-source setup/nextcloud.sh
+
+if [ "${DISABLE_NEXTCLOUD}" == "0" ]; then
+	echo Skipping Nextcloud installation
```

Comment:
Possibly change this to:

echo "Nextcloud installation was skipped due to configuration choice"

or something similar.  My reasoning is because "Skipping Nextcloud installation" alone doesn't explain why it is skipping it.